ar_exercise/exercise.rb

```diff
@@ -33,14 +33,28 @@ class Apartment < ActiveRecord::Base
 # example: get every tenant in the DB
 all_tenants = Tenant.all
 
+binding.pry
 # get the first tenant in the DB
+  #  first_tenant = all_tenants.find(0)
```

Comment:
IDs start at 1, not zero! And in fact, there's no guarantee that the first tenant has an ID of say, 12 (if the first 11 were created and then deleted).

A better way would be:

```ruby
Tenant.first
# or 
Tenant.all.first
```